.github/workflows/ruby.yml

```diff
@@ -54,15 +51,64 @@ jobs:
 
           core.setOutput('version', gemVersion);
 
-    - name: Set up Ruby 2.6
+  plantuml:
+    needs: version
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Java
+      uses: actions/setup-java@v1
+      with:
+        java-version: 14
+
+    - name: Cache Maven dependencies
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-maven-
+
+    - name: Version stamp
+      run: mvn versions:set -DnewVersion=${{ needs.version.outputs.version }}
```

Comment:
The fact that it's `-DnewVersion` infuriates me a bit, but the command seems correct from a fast search for the syntax.

Reply:
I agree the syntax is infuriating.